### DISCUSSION TITLE
Rebalance the enemy units

### DISCUSSION THREAD

**Discussion by Crizzen** [Keriew/augustus#1098] [Ideas]
Now that the power curve has moved in favour of the player (4 extra legions, additional defensive options), can I suggest some tweaks to enemy units and defensive abilities.  Not sure what is possible, so I've put a variety of ideas below.

- change unit stats so each enemy unit & civ is properly differentiated now, instead of having mostly identical stats currently (e.g. Picts should be harder than Gauls as they were never conquered, Pergamum archers should be stronger as currently spawning with 70% archers trivialises their invasions, Goths should have stronger cavalry, and so on)
- Add ability for invasions to spawn a third unit type.  Currently you only ever need to build two types of legions as enemies only spawn with 2 of (archers, cavalry, infantry, special).  Even if there is no unit art for e.g. Carthaginian Archers, you could use pergamum archers and handwave it away as "mercenaries" 
- strengthen walls. I suggest maybe +50% to +100% hp as it doesn't currently seem "fair" that an infantry unit can destroy walls in a few seconds.  Strengthening the stone walls would also help differentiate vs palisades. 
- allow multiple enemy types per map (e.g. Carthaginians & Numidians)
- add bonuses to Roman cavalry damage & defence when moving to attack (the opposite of legions who get a defence bonus for standing still)
- add javelin unit defence when moving
- Tweak projectile damage on a per-civilisation basis (e.g. boost Pergamum archers)
- Caesar sends cavalry and javelins (maybe 10% each) in addition to legions

Edit:  I would probably add, most of the above changes would make invasions *weaker* overall (with rare exceptions for enemy civs that got buffed), but I do think they'd be great for flavour.  Compensate with higher invasion multipliers perhaps.

**Comment by MSTVD** [comment]
Funny, most, if not all of what you listed is something we've considered for the past 2 years for the military additions

**Comment by Kapusha1986** [comment]
Althought Caesar legions and envading enemies both are eager to destroy your city, they can NOT be allies! So they still should fight when they meet. Just now my city was levelled by Carthaginians and Roman soldiers which were marching shoulder to shoulder..

**Comment by SuriViruS** [comment]
Some of my points have already been made above, so I didn't want to start a new thread.
I just want to elaborate on two:

- The Walls you can build are too weak. Palisades are like they don't exist and are brought down by footmen in like half a second per layer. The same footmen take down brick walls in 1 second per layer. I didn't meassure the exact time and compared it to ingame time. But just from my feeling it's a waste of ressources, at the current state, to build walls and to have them as barriers.

- The elephants the cathargians ingame send represent the mythical version of elephants, that Hannibal actually never comanded (African forest elephants or African savannah elephants). The elephant species that Hannibal commanded were smaller (North african elephants).
Anyway, the ingame elephants are too hard to kill.
If in reality you had 16 roman pilums (full legion of Auxiliaries ingame are 16 soldiers) thrown at one of the bigger elephants and lets assume 50% miss, the remaining 8 pilums kill the animal.
Ingame i have to attack and retreat with a full legion of Auxiliaries about 20 times before one elephant goes down.

I know CIII is just a game and the RTS part is not very sophisticated, but I think it could be easily improved.
From my personal experience I would strenghten all walls by the factor 10 and decrease hitpoints of the elephants at least by half..
That's all folks. Thanks for your work.

**Reply by SuriViruS** [reply]
I meant Hannibal the Carthagian, not Alexander the Great ofc.

**Reply by Miguelinileugim** [reply]
You know making all walls vastly stronger but also vastly more expensive would be an interesting addition. Same for towers and such, and would give the watchtowers (with a slight increase in price imo) some purpose.
